packages/fuels-core/src/utils/constants.rs

```diff
@@ -11,3 +11,8 @@ pub const BASE_ASSET_ID: AssetId = AssetId::BASE;
 // ANCHOR_END: default_call_parameters
 
 pub const DEFAULT_GAS_ESTIMATION_TOLERANCE: f64 = 0.2;
+
+// Supports 10 signatures
+pub const DEFAULT_SCRIPT_WITNESS_LIMIT: u64 = 720;
```

Comment:
Is there a constant on the fuel-core size that defines the size of the signature?
It would be nice to have something like 
```rust
pub const DEFAULT_SCRIPT_WITNESS_LIMIT: u64 = 10 * SIG_SIZE;
```